test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenerator/SwaggerGeneratorTests.cs

```diff
@@ -5,6 +5,7 @@
 using System.Threading.Tasks;
 using System.Reflection;
 using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
```

Comment:
The duplicate on line 14 needs removing.

Reply:
🤦 fixed